source/include/tracking/cs_TrackedDevices.h

```diff
@@ -65,6 +65,19 @@ class TrackedDevices: public EventListener {
 	uint16_t ticksLeftSecond = TICKS_PER_SECOND;
 	uint16_t ticksLeftMinute = TICKS_PER_MINUTES;
 
+	/**
+	 * Assert that TrackedDevice has standard layout (being laid out in memory exactly like it
+	 * would be in C).
+	 */
+	static_assert(std::is_standard_layout<TrackedDevice>::value,
+			"TrackedDevice has to provide standard layout");
+
+	/**
+	 * Assert that TrackedDevice is trivially copyable (basically, memcpy works).
+	 */
+	static_assert(std::is_trivially_copyable<TrackedDevice>::value,
+			"TrackedDevice has to be trivially copyable");
+
```

Comment:
As far as I am aware there is no code depending on wether or not TrackedDevice is trivially copyable or standard layout. Why assert on it?

Reply:
> Tested:
> 
> ```
> cmake .. -DPYTHON_SETUP_VENV=ON -DBOARD_TARGET=my_board
> cd ./my_board
> make python_setup_venv
> source $REPO/tools/python/crownstone_env/bin/activate```
> 
> This indeed installs and activates a venv.
> 
> Questions, notes, etc.:
> - I mis an option for automatically installing the crownstone-python-lib-* libraries. (Either editable from custom locations for development or default pip releases for 'end'-users) This could be a make target if you wish.
> - Do all board configs install a venv in the same location? Is this intended?
> - I'm missing documentation on these build/install/config features :-)
> ```

+ There's now an option for installing those libraries. It is indeed a make target.
+ To have the virtualenv different for each $target means work each time you change targets. If we want to provide more flexibility I would add a virtualenv path parameter that can be configured by the user. Then they can change it independent on target.
+ Documentation has been added.

I'd consider this hereby addressed for now.

Reply:
> As far as I am aware there is no code depending on wether or not TrackedDevice is trivially copyable or standard layout. Why assert on it?

As far as I know we want these structs to be like that, even if we don't **explicitly** make use of that. From what I understand we have these packed structs so we can use them verbatim over the wire (either dropping them in a buffer for Bluetooth messages or for UART) or on disk (in `fstorage`). 

Note, it can be probably fine-tuned, there's `is_trivially_copy_constructible`, etc. I was just intending `is_pod`, but that will be deprecated with C++20.

Reply:
I might interpret your suggestion as that actually only `register_tracked_device_packet_t` should be a pod and that the classes surrounding that are only packed to save RAM (rather than having to correspond to something on another location). However, if that's the case I would then suggest to start writing down which structs should be considered pods and which aren't, because now it's unclear to a (new) developer.

Reply:
Asserts have been removed again. The packed property is not used outside of these classes.